README.md

```diff
@@ -50,7 +50,7 @@ visit the [IBM Quantum Platform](https://quantum.ibm.com/).
 Once you have invoked the `poetry shell` command as described above, you can dispatch a job by specifying the parameters of the job you wish to launch in a configuration file. The following dispatches a job on the ibm-strasbourg device for quantum volume.
 
 ```
```

Comment:
[nit] Use `sh` after the three ` chars to syntax-highlight (same for the `poll` example below).